package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ls1intum/apollon",
-  "version": "2.9.1",
+  "version": "2.9.1-beta.0",
```

Comment:
Any specific reason for the version change?

Reply:
I created a beta release from this branch.  Will be adjusted